pkg/network/setup/netpod/netpod_test.go

```diff
@@ -1688,9 +1695,16 @@ func expectedDHCPConfig(podIfaceCIDR, podIfaceMAC, defaultGW, staticRouteDst str
 	if err != nil {
 		return nil, err
 	}
+
+	staticRouteToWiderSubnetDest, err := vishnetlink.ParseAddr(staticRouteToWiderSubnet)
+	if err != nil {
+		return nil, err
+	}
+
 	routes := []vishnetlink.Route{
 		{Gw: net.ParseIP(defaultGW)},
 		{Dst: destAddr.IPNet, Gw: net.ParseIP(defaultGW)},
+		{Dst: staticRouteToWiderSubnetDest.IPNet},
```

Comment:
This is odd. What is the meaning of a route without a GW/next-hop?

Reply:
The GW is implicitly set to `nil`.
When the route's Gw is set to `nil` the DHCP server sends "0.0.0.0" as the gateway:
https://github.com/kubevirt/kubevirt/blob/12c11b1808703367a8cdef54ce8839197def26a1/pkg/network/dhcp/server/server.go#L270-L273

Which according to RFC 3442[1] is the expected behavior when the link in question is directly connected to the wider subnet.

I've explicitly set the Gw to `nil`.

[1] https://www.rfc-editor.org/rfc/rfc3442.html